readme.md

```diff
@@ -1,139 +1,47 @@
-# Utreexo
+# UTreeXO
 
-A dynamic hash based accumulator designed for the Bitcoin UTXO set
-
-Check out the ePrint paper here: [https://eprint.iacr.org/2019/611](https://eprint.iacr.org/2019/611)
+![utreexo cover image](/docs/assets/utreexo.png)
 
-Currently under active development.  If you're interested and have questions, checkout #utreexo on irc.libera.chat.
+Utreexo is a novel hash-based dynamic accumulator, which allows the millions of unspent outputs to be represented under a kilobyte -- small enough to be written on a sheet of paper. There is no trusted setup or loss of security; instead, the burden of keeping track of funds is shifted to the owner of those funds.
 
-Logs for libera are [here](http://gnusha.org/utreexo/)
+Check out the ePrint paper here: https://eprint.iacr.org/2019/611
 
----
-## Importing Utreexo
-The raw accumulator functions are in package accumulator. This is a general accumulator and is not bitcoin specific. For Bitcoin specific accumulator, look to the csn and bridgenode packages.
+Currently, transactions specify inputs and outputs, and verifying an input requires you to know the whole state of the system. With Utreexo, the holder of funds maintains a proof that the funds exist and provides that proof at spending time to the other nodes. These proofs are compact (under 1KB) but represent the utreexo model’s main downside; they present an additional data transmission overhead that allows a much smaller state.
```

Comment:
Sometimes the proofs are big. The accumulator state (under a kilobyte) is small but proofs can get big so I'd get rid of the part that says "These proofs are compact (under 1KB)".